library/src/main/java/com/schibsted/spain/barista/custom/DisplayedMatchers.java

```diff
@@ -28,4 +28,9 @@
   public static Matcher<View> displayedWithText(String text) {
     return allOf(isDisplayed(), withText(text));
   }
+
+  @NonNull
+  public static Matcher<View> displayedAssignableFrom(final Class<? extends View> clazz) {
+    return allOf(isDisplayed(), isAssignableFrom(clazz));
```

Comment:
Suggestion: A nice trick to avoid the ugly `clazz` hack is telling what it's for. Like `viewClass`.